packages/expo-av/ios/EXAV/EXAVPlayerData.m

```diff
@@ -261,6 +264,12 @@ - (void)setStatus:(NSDictionary *)parameters
     NSNumber *rate = parameters[EXAVPlayerDataStatusRateKeyPath];
     _rate = rate;
   }
+  
+  if ([parameters objectForKey:EXAVPlayerDataStatusPitchCorrectionQualityKeyPath] != nil) {
```

Comment:
```suggestion
  if (parameters[EXAVPlayerDataStatusPitchCorrectionQualityKeyPath] != nil) {
```

Reply:
Wanted to be consistent with other similar code in this file. Should I change this one line?

Reply:
yes, please 🙂 we want to make things more consistent across all classes and not just this one